package.json

```diff
@@ -187,17 +187,17 @@
     "@zeit/dockerignore": "0.0.3",
     "JSONStream": "^1.0.3",
     "archiver": "^3.1.1",
-    "balena-config-json": "^4.0.0",
+    "balena-config-json": "^4.1.0",
     "balena-device-init": "^5.0.2",
-    "balena-errors": "^4.3.0",
-    "balena-image-manager": "^7.0.1",
-    "balena-preload": "^10.2.0",
+    "balena-errors": "^4.4.1",
+    "balena-image-manager": "^7.0.3",
+    "balena-preload": "^10.2.3",
     "balena-release": "^3.0.0",
-    "balena-sdk": "^14.8.0",
-    "balena-semver": "^2.2.0",
+    "balena-sdk": "^15.2.1",
+    "balena-semver": "^2.3.0",
     "balena-settings-client": "^4.0.5",
-    "balena-settings-storage": "^6.0.0",
-    "balena-sync": "^11.0.0",
+    "balena-settings-storage": "^6.0.1",
```

Comment:
Small fix to add a missing dep, not strictly required but will avoid any issues if for some reason we don't install the dep/install a different version of it